pkgs/stdenv/freebsd/bootstrap-files/x86_64-unknown-freebsd.nix

```diff
@@ -0,0 +1,13 @@
+{
+  unpack = import <nix/fetchurl.nix> {
+    url = "http://192.168.122.1:8000/result/on-server/unpack.nar.xz";
+    hash = "sha256-y6quCU9JKnKBdHDcUkdkM0ypWDT2cdSiqR1WqA+8ozE=";
+    name = "boostrapUnpacked";
```

Comment:
the name should match the nar filelname stripped of `.nar.xz` so the refresh-tarballs.bash script works correctly.
https://github.com/NixOS/nixpkgs/blob/f1550ab08e96429fd028ff6b828a3bf2f2d2f1db/maintainers/scripts/bootstrap-files/refresh-tarballs.bash#L252

Reply:
afaik refresh-tarballs.bash doesn't parse this file and just generates it. this attribute is totally unused outside of just generating the exact store path, which will just be referred to by a variable derived from this expression anyway.

Reply:
that's correct. i was confused, the name of the store path shouldn't matter.